plugins/chaos/torpedo_chaos/log_analyzer.py

```diff
@@ -93,12 +94,33 @@ def log_analyzer(log_dir, service_list):
                                           'utf-8').strip("\n")
         result_dict = {
                        "Test Case": svc['service'],
+                       "Cluster Name": svc['cluster-type'],
                        "TimeStamp": cur_time,
                        "Duration": svc['duration'],
                        "Number of pods killed": kill_tc,
                        "Pass test count": pass_tc,
                        "Fail test count": fail_tc
                       }
+        #push Test_results to elasticsearch
+        if 'elasticsearch' in svc:
```

Comment:
Can we create a separate function and put this code under that function?

Reply:
Yes Possible, I will do that